src/content/docs/accounts/accounts/account-maintenance/promotions-ui.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Product promotions 
+tags:
+  - Accounts
+  - Accounts and billing
+  - General account settings
+metaDescription: 'How to sign up for product promotions'
+freshnessValidatedDate: 2024-07-15 
+---
+
+
+import accountsPreviewsandTrials from 'images/accounts_screenshot-crop_previews-and-trials.webp'
+
+import accountsPromotionsTable from 'images/accounts_screenshot-crop_promotions-table.webp'
+
+import accountsManagePreview from 'images/accounts_screenshot-crop_manage-preview.webp'
+
+import accountsPromoUpgradeButton from 'images/accounts_screenshot-crop_promo-upgrade-button.webp'
+
+
+
+Promotions are special offers to try out product features at New Relic. You can browse through our various promotions and enroll through our promotions management UI. 
+
+The type of promotion currently available is the product preview. This is the opportunity to try out a feature before it is released (pre-GA). The promotions management UI gives you:
```

Comment:
This sentence is a essentially saying the same thing as the first sentence. Consider consolidating.

Reply:
@ally-sassman I'm looking to the future when we'll offer both previews (pre-GA features) and trials (GA features). So, this is what I'm trying to communicate:

Line 22: This defines a promotion as a way to try out a feature (keeping this larger category of promotions that will eventually include trials, which are somewhat different.

Line 24: This explains that we are currently only offering one type of promotion (without a reference to trials). Then, we clarify that a preview is a chance to try out a feature before it is GA.

These may be somewhat similar, but once we start offering trials, it will become easier to differentiate the two.